src/renderers/WebGLRenderer.js

```diff
@@ -489,7 +489,7 @@ class WebGLRenderer {
 
 			}
 
-			state.viewport( _currentViewport.copy( _viewport ).multiplyScalar( _pixelRatio ).floor() );
+			state.viewport( _currentViewport.copy( _viewport ).multiplyScalar( _pixelRatio ).round() );
```

Comment:
In at least six other instances in the code, `floor()` is still used when applying `pixelRatio`. Was this an oversight, or was this intentional?

/ping @Mugen87
/ping @gkjohnson

Reply:
To me that is an oversight. When looking at the code, the usage in `setRenderTarget()` should be updated to `round()` as well.

https://github.com/mrdoob/three.js/blob/0bc990f5f728552c6cc3931d1b44afa9718aef40/src/renderers/WebGLRenderer.js#L2286-L2287

Not sure about `getDrawingBufferSize()` though.

https://github.com/mrdoob/three.js/blob/0bc990f5f728552c6cc3931d1b44afa9718aef40/src/renderers/WebGLRenderer.js#L412

Depending on how we decide, the code in `examples/jsm/renderers/common/Renderer.js` should be updated as well.